contracts/StreamManager.sol

```diff
@@ -280,4 +285,12 @@ contract StreamManager is IStreamManager, ReentrancyGuard {
         payer = _payer;
         emit PayerChanging(_payer);
     }
+
+    ///@dev changing address of the fee
+    function chaingeAddressFee(address _feeAddress) public onlyAdmin {
```

Comment:
```suggestion
    function changeComissionAddress(address _feeAddress) public onlyAdmin {
```

- update in other places where necessary. Also fixed a typo. It's not `chainge` it's `change`
- new name suggested by Vadim